bundles/certified-operators/5.0.7/manifests/minio-operator.clusterserviceversion.yaml

```diff
@@ -86,7 +86,7 @@ metadata:
     operators.operatorframework.io/builder: operator-sdk-v1.22.2
     operators.operatorframework.io/project_layout: unknown
     repository: https://github.com/minio/operator
-    containerImage: quay.io/minio/operator:v5.0.7
+    containerImage: quay.io/minio/operator@sha256:32527ce94bf19b19e3fe35acfc2ce57ad1d94d8280083d94dc1d9d45a365763a
```

Comment:
why are we going back and forth on this, what is the correct way? @cniackz @dvaldivia @pjuarezd ?

Reply:
Redhat requires to set the `containerImage` in the sha256 format to certify the Operator, so this is a requirement.

We publish Operator as follows:

Operator release PR -> merge -> Build and publish container image to registry (quay.io) -> update manifest with image sha256 from registry -> publish to OperatorHub

Now, we cannot know the container image sha256 before the image is pushed to quay.io, that is why we have this process split in two scripts `olm.sh` and olm-post-scrip.sh`.

I don't think this is wrong per se, just inconveniently implies two PR's at different moments.

Reply:
If by do it the right way you mean, not have several PR's modifying the OperatorHub manifests each release, I think we could do it several ways:

1) Create OperatorHub manifests only after the container image is published.
However it still requieres two PR's One for the Operator Release version update and other to create the OperatorHub manifests.

2) Do not save the OperatorHub manifests at all in minio/operator repo.

We don't rally need those manifest in this repo, they do nothing but to be kept as a backup/copy. The manifest that actually have an effect are the ones in the redhat repos: https://github.com/redhat-openshift-ecosystem/certified-operators/ and https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators.

3) We could automate the OperatorHub manifests creation via github actions when a release is published, and save those on another repo, either in a clone of the https://github.com/redhat-openshift-ecosystem/certified-operators/ and https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators repos, or a dedicated repo.

Reply:
PR;s to publish on redhat marketplace and certified operators
https://github.com/redhat-openshift-ecosystem/certified-operators/pull/2656
https://github.com/redhat-openshift-ecosystem/redhat-marketplace-operators/pull/527